code/modules/projectiles/guns/energy/energy.dm

```diff
@@ -23,6 +22,15 @@
 /obj/item/gun/energy/gun/cyborg/emp_act()
 	return
 
+/obj/item/gun/energy/gun/examine_more(mob/user)
+	. = ..()
+	. += "A hybrid fire laser gun designed and patentend by Shellguard Munitions Co. Initally designed after market demand for a versatile weapon that merges disabler and laser functions without the need for switching during active threats. \
+	This would cause the company to experiment and produce a weapon to fit between Shellguards own disabler, and LG series weapons. This led to the creation of the EG series, positioning the EG-7 as a flagship model intended to eventually replace Shellguard's disabler pistols and LG series weapons. \
```

Comment:
```suggestion
	This would cause the company to experiment and produce a weapon to fit between Shellguard's own disabler, and LG series weapons. This led to the creation of the EG series, positioning the EG-7 as a flagship model intended to eventually replace Shellguard's disabler pistols and LG series weapons. \
```